### PR TITLE
Get rid of inconsistency with neode setup

### PR DESCRIPTION
**Pull request by Tirokk**
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-02T23:23:55Z" title="Tuesday, December 3rd 2019, 12:23:55 am +01:00">Dec 3, 2019</time>_
_Merged <time datetime="2019-12-10T10:04:36Z" title="Tuesday, December 10th 2019, 11:04:36 am +01:00">Dec 10, 2019</time>_
---

## 🍰 Pullrequest

    Implement getDriver() and getNeode() the same way

    I want to introduce `neode` and `neo4j-graphql-js` to my students and
    saw this.
------
    Fix cypress tests

    Since `dotenv` in the backend now loads configuration from a file
    relative to the current file, we don't have to duplicate the neo4j
    configuration settings here.

### CODE REVIEW

**Comment by Tirokk** [comment]
> [cypress[bot]](https://github.com/apps/cypress) **Authored by [cypress[bot]](https://github.com/apps/cypress)**
_<time datetime="2019-12-02T23:50:45Z" title="Tuesday, December 3rd 2019, 12:50:45 am +01:00">Dec 3, 2019</time>_

---



<p align="center">
  <br />
  
</p>

---

### Test summary
<a href="https://dashboard.cypress.io/projects/qa7fe2/runs/2810/specs"></a> <b title="Passed">48</b>  &bull; <a href="https://dashboard.cypress.io/projects/qa7fe2/runs/2810/failures"></a> <b title="Failed">0</b>  &bull;  <b title="Pending">0</b> &bull;  <b title="Skipped">0</b>

---

### Run details
| | |
| --- | --- |
| **Project** | Human-Connection |
| **Status** | Passed |
| **Commit** | a3251710fa |
| **Started** | Dec 10, 2019 8:58 AM |
| **Ended** | Dec 10, 2019 9:12 AM |
| **Duration** | 14:00 <a href="https://on.cypress.io/parallelization" title="Enable parallelization to reduce run duration">💡</a> |
| **OS** | Linux Ubuntu Linux - 16.04 |
| **Browser** | Chromium 78  |

[View run in Cypress Dashboard ➡️](https://dashboard.cypress.io/projects/qa7fe2/runs/2810/specs)



---

This comment has been generated by **cypress-bot** as a result of this project's GitHub integration settings. You can manage this integration in [this project's settings in the Cypress Dashboard](https://dashboard.cypress.io/projects/qa7fe2/settings)

File: backend/src/middleware/hashtags/hashtagsMiddleware.spec.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ beforeAll(() => {
     context: () => {
       return {
         user: authenticatedUser,
-        neode: instance,
+        neode,
```
Comment (Tirokk) [inline backend/src/middleware/hashtags/hashtagsMiddleware.spec.js:39]: > [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-06T09:15:44Z" title="Friday, December 6th 2019, 10:15:44 am +01:00">Dec 6, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -36,7 +36,7 @@ beforeAll(() => {
     context: () => {
       return {
         user: authenticatedUser,
-        neode: instance,
+        neode: neode,
```


```suggestion
        neode,
```
can't we use ES6 syntax here?

File: backend/src/schema/resolvers/users/blockedUsers.spec.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ beforeEach(() => {
       return {
         user: authenticatedUser,
         driver,
-        neode: instance,
+        neode,
```
Comment (Tirokk) [inline backend/src/schema/resolvers/users/blockedUsers.spec.js:23]: > [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-06T09:17:11Z" title="Friday, December 6th 2019, 10:17:11 am +01:00">Dec 6, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -20,7 +20,7 @@ beforeEach(() => {
       return {
         user: authenticatedUser,
         driver,
-        neode: instance,
+        neode: neode,
```


```suggestion
        neode,
```
same as :point_up_2:

**Review by Mogge — CHANGES_REQUESTED** [review]
Looks good, just requested two small changes @roschaefer

**Review by Mogge — APPROVED** [review]

**Pull request closed without merging**